daemon/algod/api/server/v2/dryrun.go

```diff
@@ -135,13 +135,11 @@ func (ddr *dryrunDebugReceiver) updateScratch() {
 	}
 
 	if any {
-		if ddr.scratchActive == nil {
-			ddr.scratchActive = make([]bool, maxActive+1, 256)
-		}
-		for i := len(ddr.scratchActive); i <= maxActive; i++ {
+		ddr.scratchActive = make([]bool, maxActive+1, 256)
+		for i := 0; i <= maxActive; i++ {
```

Comment:
creating a new slice and iterating starting from 0 is less efficient but seems much easier for me to understand